views/thingpedia_dev_oauth.pug

```diff
@@ -0,0 +1,59 @@
+extends layout
+
+block styles
+  link(rel='stylesheet', href='/stylesheets/thingpedia.css')
+  link(rel='stylesheet', href='/stylesheets/thingpedia_dev_sidebar.css')
+
+block page_name
+  - stdlayout = false;
+  - var section = 'oauth';
+  = _("Thingpedia Developer Portal")
+
+block content
+  div.container-fluid#page-body
+    div.row
+      div.col-lg-3.col-md-4
+        include thingpedia_dev_sidebar
+
+      div.col-xs-12.col-md-8.col-lg-9
+        if authenticated && user.developer_org !== null
+          section.divider
+            h3= _("Your OAuth 2.0 Applications")
```

Comment:
Maybe it's time to drop the 2.0 everywhere, and just go with OAuth Applications?